README.md

```diff
@@ -1,13 +1,20 @@
-# jkennedy-research-and-reflection
+# Jacob Kennedy - Research & Reflection
 
+## Course goals
+
+<!-- Context 1 -->
 Over the time in this course, I want to take actionable steps to become a stronger programmer and more in tune and efficient with what I do when developing. I often write code and create programs and find myself kicking myself when I see others do the same tasks so much more efficiently than I. I want to work towards having a deep understanding of the effects of what I'm doing when writing programs and how I interact with the machine I work on. 
 
-I'm going to spend time on coding websites and online communites challenging myself to write more complex and meaningful code on a daily basis. I want to spend at least 8 hours a week focusing specifically on programming in Java.
+<!-- Goal 1 -->
```

Comment:
This goal checks a lot of the boxes. You've been specific in your actions (write Java code based on online community participation) and you've made it achievable by stating a weekly time allotment. We'll want to get a bit more specific as we revise this goal in a few weeks, but what I'm most concerned with right now is the measure. 

You say you want to write "more complex and meaningful code". What does that mean to you right now and how will you know you've achieved it? I can think of a couple ways you might do this, but I'd like you to take a shot at it first. You can think of this in either the short (i.e. matter of weeks) or long term (end of the course). 

Edit your README.me to reflect your thoughts on the measure.